src/Config.php

```diff
@@ -219,14 +218,23 @@ public function get($path, $default = null)
      *
      * @return mixed
      */
-    protected function doReplacements($value)
+    public function doReplacements($value = null)
```

Comment:
Just flagging before tonight's meeing… but if we're going to add this to the API, can we at least mark it `@internal` as there is a :dart: :gun:  on the `Config` class for ^3.1